src/view/com/util/List.tsx

```diff
@@ -37,7 +40,8 @@ function ListImpl<ItemT>(
   const isScrolledDown = useSharedValue(false)
   const contextScrollHandlers = useScrollHandlers()
   const pal = usePalette('default')
-
+  const showsVerticalScrollIndicator =
+    !useGate('hide_vertical_scroll_indicators') && isWeb
```

Comment:
should this be && or should it be || ?

Reply:
nm that's correct

Reply:
wait. is it?

Reply:
God I had my explanation typed out and then realized you're right. What a mind fuck. Sec.

Reply:
okay fixed...

Case 1:
- Gate false -> inverted -> true
- Is not on web -> false
- Shows indicator

Case 2:
- Gate false -> inverted -> true
- Is on web -> true
- Shows indicator

Case 3:
- Gate true -> inverted -> false
- Is not on web -> false
- Doesn't show

Case 4:
- Gate true -> inverted -> false
- Is on web -> true
- Shows indicator